build/web-client-rpc.md

```diff
@@ -0,0 +1,55 @@
+
+### TL;DR
+The Nimiq Web Client establishes direct connections with other peers in the network, making it a part of the blockchain network. In contrast, the RPC follows a client-server model, where clients connect to a single server that interacts with multiple peers on the network.
+
+<img class="object-contain max-h-[max(60vh,180px)]" src="/assets/images/protocol/network.png" alt="skip block struct" />
+
+## Nimiq Web Client
+
+The [Nimiq Web Client](/build/web-client/getting-started.md) is a JavaScript library that enables direct participation in the Nimiq PoS ecosystem directly from your web browser. This client is compiled from Rust to WebAssembly and offers a lightweight solution for users who want to interact with the Nimiq blockchain.
+
+The Web Client provides a user-friendly interface for building consensus with other nodes, creating wallets, sending transactions, and interacting with the blockchain with simple operations. It simplifies user interaction and offers easy integration.
+
+Users can access the Web Client on mobile devices, such as smartphones and tablets, providing flexibility and convenience for managing their Nimiq wallets and transactions on the go. The Web Client is designed to be compatible with major web browsers, including Google Chrome, Mozilla Firefox, and Microsoft Edge, ensuring a consistent user experience across different platforms.
+
+## Nimiq RPC
+
+The [Nimiq RPC](/build/rpc-docs/index.md) provides developers access to a node that is part of the Nimiq blockchain and establishes a connection between a client and a single server, which then interacts with multiple peers on the network.
+
+The RPC offers more elaborate functionalities, such as retrieving information about the blockchain state, submitting transactions, managing accounts, and configuring node settings. It is particularly valuable for developers and nodes who require advanced control and automation of blockchain operations.
+
+The RPC supports interaction with nodes and allows users to call its endpoints using JavaScript, shell scripting languages, or any language or software capable of sending requests according to the JSON-RPC specification. Comprehensive API documentation is available for the RPC, detailing the methods, parameters, and endpoints supported by the interface, facilitating seamless integration and development.
+
+
+## How to know which one to use?
+
+| Aspect | Web Client | RPC Client |
+| --- | --- | --- |
+| Nimiq Use | Nimiq Pay App | TrustWallet |
+| Accessibility | Requires familiarity with JavaScript and similar frameworks like NodeJS; can be run in a browser | Requires adherence to the JSON-RPC specification |
+| Functionality | Basic blockchain tasks | Gives detailed control over blockchain functions |
```

Comment:
I wouldn't call it "basic" what the web client can do. It can interact with the blockchain just like you can with RPC. You can send all types of transactions and do with your accounts everything possible in the protocol. RPC just gives you control over the _node_, too. And since the node is usually a full or history node, it has more options than the web-client's light node type.